js/index.js

```diff
@@ -0,0 +1,183 @@
+    list1: 'HTML/CSS',
+    list2: 'Ruby on Rail',
+    list3: 'Javascript',
+    p: 'see this project',
+  },
+  {
+    seeProjectBtn: document.getElementById('project-btn3'),
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt explicabo, quidem dolorum totam consequuntur vitae consectetur, odio asperiores voluptas magni tenetur vel. Accusamus ut, minima quas aliquid consectetur cumque molestiae at modi fugit dolores ipsum, perferendis a quam saepe et doloremque quae reiciendis id, assumenda rerum. Fuga voluptatum earum aliquam?',
+    image: document.createElement('IMG'),
+    image_src: document.getElementById('project_img3').src,
+    list1: 'HTML/CSS',
+    list2: 'Ruby on Rail',
+    list3: 'Javascript',
+    p: 'see this project',
+  },
+  {
+    seeProjectBtn: document.getElementById('project-btn4'),
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt explicabo, quidem dolorum totam consequuntur vitae consectetur, odio asperiores voluptas magni tenetur vel. Accusamus ut, minima quas aliquid consectetur cumque molestiae at modi fugit dolores ipsum, perferendis a quam saepe et doloremque quae reiciendis id, assumenda rerum. Fuga voluptatum earum aliquam?',
+    image: document.createElement('IMG'),
+    image_src: document.getElementById('project_img4').src,
+    list1: 'HTML/CSS',
+    list2: 'Ruby on Rail',
+    list3: 'Javascript',
+    p: 'see this project',
+  },
+  {
+    seeProjectBtn: document.getElementById('project-btn5'),
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt explicabo, quidem dolorum totam consequuntur vitae consectetur, odio asperiores voluptas magni tenetur vel. Accusamus ut, minima quas aliquid consectetur cumque molestiae at modi fugit dolores ipsum, perferendis a quam saepe et doloremque quae reiciendis id, assumenda rerum. Fuga voluptatum earum aliquam?',
+    image: document.createElement('IMG'),
+    image_src: document.getElementById('project_img5').src,
+    list1: 'HTML/CSS',
+    list2: 'Ruby on Rail',
+    list3: 'Javascript',
+    p: 'see this project',
+  },
+  {
+    seeProjectBtn: document.getElementById('project-btn6'),
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nesciunt explicabo, quidem dolorum totam consequuntur vitae consectetur, odio asperiores voluptas magni tenetur vel. Accusamus ut, minima quas aliquid consectetur cumque molestiae at modi fugit dolores ipsum, perferendis a quam saepe et doloremque quae reiciendis id, assumenda rerum. Fuga voluptatum earum aliquam?',
+    image: document.createElement('IMG'),
+    image_src: document.getElementById('project_img6').src,
+    list1: 'HTML/CSS',
+    list2: 'Ruby on Rail',
+    list3: 'Javascript',
+    p: 'see this project',
+  },
```

Comment:
- Hi again, @Mylo16 :smile:
- [ ] I'm if this looks disturbing to you, but it really shouldn't. Like I stated the other time, it's a mandatory requirement to ensure that for this project submission, you don't have much hardcoded lines in your `index.html` to render the project section. This will make the process of adding more projects into the portfolio easier and faster in the future. To further explain this, I shared a repository with but it seems that it's probably not to clear to navigate but as requested, please include the absolute values of the key pairs instead of querying the DOM - `document.createElement()` or `dcocument.getElementById()`. Here's a more precise format that won't break up your code too much:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"], # instead of list1, list2, list3 [OPTIONAL]
    link: "link_to_live_version.com",
}]
```